content/open_science/resources.md

```diff
@@ -0,0 +1,149 @@
+
+{{< image_float 
+  image="/img/open_science/resources/coara.png" 
+  alt="COARA logo"
+>}}
+
+On 23 February 2024, SciLifeLab announced that it had joined [CoARA](https://coara.eu/) (Coalition for Advancing Research
+Assessment) as a signatory member. This reaffirmed SciLifeLab’s commitment to advancing research assessment and open
+science practices.
+You can review SciLifeLab’s CoARA Action Plan [here](https://docs.google.com/document/d/1KAKOK6HoXsixQA5FaPegDIw1w3uI6OvlHgT2YR1m0pg/edit?tab=t.0).
+Note, [DORA](https://sfdora.org) (San Francisco Declaration on Research Assessment) laid
+the foundation for research assessment reform by advocating for qualitative evaluation over journal-based metrics,
+a principle that CoARA builds upon to drive systemic change through collective institutional commitment.
+
+{{< /image_float  >}}
+
+#### SciLifeLab Data Policy
+
+The [SciLifeLab Data Policy](https://www.scilifelab.se/wp-content/uploads/2023/10/Data-Policy.pdf) outlines SciLifeLab's
+commitment to Open Science, research transparency, and the FAIR
+principles, ensuring that data from publicly funded research is as open as possible, while adhering to legal and
+ethical requirements. It sets expectations for SciLifeLab-affiliated programs, infrastructure, and researchers to promote
+data sharing, reproducibility, and accessibility, while providing the necessary resources, support, and governance to
+integrate these principles into the research we produce.
+
+#### SciLifeLab Training Hub
+
+The [SciLifeLab Training Hub](https://www.scilifelab.se/training/) offers a range of educational opportunities and
+training infrastructure, to advance Open Science and FAIR learning, ensuring Sweden’s life science community has open
+access to world-class knowledge and expertise. View [SciLifeLab Training Hub’s website](https://www.scilifelab.se/training/)
+to learn more about training tools, upcoming courses and ways in which the Training Hub team can support you.
+
+#### SciLifeLab RDM Guidelines
+
+{{< image_float 
+  image="/img/open_science/resources/rdm_guidelines.png" 
+  alt="RDM Guidelines logo"
+>}}
+
+In collaboration with [NBIS](https://nbis.se/), SciLifeLab Data Centre has developed the
+[Research Data Management Guidelines](https://data-guidelines.scilifelab.se/) as a knowledge
+hub to support Swedish life science researchers in managing their data throughout the research lifecycle. This resource
+offers guidance on key topics, including data management plans, metadata, versioning, Open Science & FAIR principles.
+
+{{< /image_float  >}}
+
+## External Resources {.highlighted-header #external-resources}
+
+{{< logo_row >}}
+/img/open_science/resources/fair_aware.png|FAIR Aware logo
+/img/open_science/resources/eu_commission.png|EU Commission logo
+/img/open_science/resources/vetenskapsrådet.png|Vetenskapsrådet logo
+/img/open_science/resources/unesco.png|UNESCO logo
+/img/open_science/resources/plan_s.png|Plan-S logo
+{{< /logo_row >}}
+
```

Comment:
You can also do the following for above

```
{{< logo_row 
"/img/open_science/resources/fair_aware.png|FAIR Aware logo"
"/img/open_science/resources/eu_commission.png|EU Commission logo"
"/img/open_science/resources/vetenskapsrådet.png|Vetenskapsrådet logo"
"/img/open_science/resources/unesco.png|UNESCO logo"
"/img/open_science/resources/plan_s.png|Plan-S logo"
>}}
```

Basically passing them as positional argument, rather than `Inner` string. Just another minor tweak in the shortcode is needed, mentioned it there.